toastify/toastify.nuspec

```diff
@@ -4,9 +4,9 @@
   <metadata>
    <id>toastify</id>
    <title>toastify</title>
-   <version>1.7.8</version>
+   <version>1.8.3</version>
    <authors>Geoffrey Huntley</authors>
-   <owners>ghuntley</owners>
+   <owners>ghuntley, slayerice09</owners>
```

Comment:
What's the problem with this? I am now one of the maintainers for toastify on chocolatey as they threatened to remove it if it didn't...